examples/kms/vault/kms-config.yaml

```diff
@@ -9,6 +9,8 @@ data:
         "vaultAddress": "http://vault.default.svc.cluster.local:8200",
         "vaultAuthPath": "/v1/auth/kubernetes/login",
         "vaultRole": "csi-kubernetes",
+        "vaultTokenAuth": "true",
```

Comment:
I agree that defaulting to `false` would be better, specially as you mentioned that this method is not suitable for production deployments. Please also add a comment in the `.yaml` expressing the recommendation to not use this functionality, except for testing.